editoast/schemas/src/rolling_stock.rs

```diff
@@ -117,6 +119,27 @@ pub struct RollingStock {
     pub other_categories: Vec<TrainMainCategory>,
 }
 
+impl RollingStock {
+    pub fn supported_signaling_systems(&self) -> Vec<String> {
+        self.supported_signaling_systems
+            .iter()
+            .map(|s| s.to_string())
+            .collect()
+    }
+
+    pub fn get_etcs_brake_params(&self) -> Option<&EtcsBrakeParams> {
+        self.supported_signaling_systems
+            .iter()
+            .find_map(|s| match s {
+                SupportedSignalingSystem::EtcsLevel2 { brake_params } => Some(brake_params),
+                SupportedSignalingSystem::BAL => None,
+                SupportedSignalingSystem::BAPR => None,
+                SupportedSignalingSystem::TVM300 => None,
+                SupportedSignalingSystem::TVM430 => None,
```

Comment:
Please group the match (resolve if linter says otherwise):
```suggestion
                SupportedSignalingSystem::BAL | SupportedSignalingSystem::BAPR | SupportedSignalingSystem::TVM300 | SupportedSignalingSystem::TVM430 => None,
```

Reply:
done in https://github.com/OpenRailAssociation/osrd/pull/13435/commits/4d0efb625492f081542d410931521701bc5cddc4